### ISSUE TITLE
Publish an empty message

### ISSUE THREAD

**Issue by adamrankin** [rosmed/slicer_ros2_module#77]
I would like to publish a std_msgs/Empty to a topic, can this be done currently?

I took a look at the macro for the other message types and it doesn't look like it would work for Empty. I can write up a PR to create the class for this message type specifically.

**Comment by adeguet1** [comment]
I just pushed a quick fix.

Adding proper support for `EmptyPublisher->Publish()` without argument is not as simple as I first thought,  It would require some re-implementation with code pulled from both the `VTK_MRML_ROS_PUBLISHER_NATIVE` macros and the templated base class  `vtkMRMLROS2PublisherNativeInternals`.  Instead I used a "pseudo" type in Slicer, i.e. `bool`.  So now the `Publish()` method expects a boolean that it just ignores.  I realize it's confusing to call something like `EmptyPublisher->Publish(true/false)`.  So I changed the `Publish` signature in the native macro to define a default argument value: `Publish(const slicer_type & value = 0)`.  I'm not totally happy with this because it would throw an exception when instantiated and called for `std::string`.

Alternatively, we could use `std::string` instead of `bool` for the dummy slicer type.  Users would have to call `EmptyPublisher->Publish("")`.   Any preference from users?  @LauraConnolly @aravindskumar98 @adamrankin
